### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by iamakulov**
* Use better English
* Add additional details about Webpack installation and Twitter/Pinterest options
* Move the Webpack section into _Setup_
* Add information about the supported browsers (#26)
* Add a screenshot

cc @ilyabirman for review, @vitkarpov if interested.

### CODE REVIEW

**Comment by iamakulov** [comment]
You can see the updated Readme here: https://github.com/ilyabirman/Likely/blob/update-readme/readme.md.

File: readme.md [hunk 1]
```diff
@@ -88,36 +73,87 @@ Likely supports following social networks:
...
+_Thanks to [Corey Maler](https://github.com/Corey-Maler) for the instructions._
 
 ## Options
 
-You can set some options on Likely button set via `data-*` attributes:
+You can configure Likely by specifying `data-*` attributes on a button group or on a button.
+
+### Common options
+
+These options should be specified on on the `div` with the `likely` class.
```
Comment (vitkarpov) [inline readme.md]: `on on`

File: readme.md [hunk 2]
```diff
@@ -1,70 +1,55 @@
...
-Then, initiate Likely:
-
-```js
-// on document ready or in the end
-require('ilyabirman-likely').initiate();
-```
-
-Instruction for using with webpack provided by @Corey-Maler.
+Or [download the repository code](https://github.com/ilyabirman/Likely/archive/master.zip) and move `release/likely.js` and 
+`release/likely.css` to a desired directory.
```
Comment (vitkarpov) [inline readme.md]: to the desired (specific) directory

Comment (vitkarpov) [inline readme.md]: I think

**Comment by vitkarpov** [comment]
👍

**Comment by vitkarpov** [comment]
cool

File: readme.md [hunk 3]
```diff
@@ -1,70 +1,55 @@
 # Likely
 
-The social sharing buttons that aren’t shabby
+> The social sharing buttons that aren’t shabby
+
+![Likely screenshot](http://i.imgur.com/KvMkD6R.png)
 
 ## Install
 
 Install via npm:
```
Comment (iamakulov) [inline readme.md]: with npm

File: readme.md [hunk 4]
```diff
@@ -1,70 +1,55 @@
...
 
 ```sh
-npm install ilyabirman-likely
+$ npm install ilyabirman-likely --save
 ```
 
 Install via bower:
```
Comment (iamakulov) [inline readme.md]: with bower

**Comment by iamakulov** [comment]
Whoops, included extra files, will re-push the commits when I get to the computer.

File: readme.md (hunk 1 above)
Comment (iamakulov) [inline readme.md]: Good catch!

**Comment by iamakulov** [comment]
Excluded the unnecessary files and fixed the notes.

**Comment by iamakulov** [comment]
@ilyabirman modified the Readme.

**Comment by iamakulov** [comment]
Updated the screenshot to include Odnoklassniki and Telegram, made it point to the test page and changed NPM to npm.

**Comment by iamakulov** [comment]
Updated the Odnoklassniki button on the screenshot. Ilya Birman approved this in the private conversation, so we’re ready to merge.

**Comment by iamakulov** [comment]
Also, changed the example text on the Odnoklassniki button from “Share” to “Like” so that it corresponds to the official Odnoklassniki’s English translation.

**Pull request merged**